src/components/AssetInfo/AssetInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { hexToNumberString } from "web3-utils";
+
+import { ETHERSCAN_BASE_URL } from "../../config";
+
+const makeEtherscanTokenURL = ({ registryAddress, tokenId }: ERC721TokenParameters) => {
+  const tokenIdDecimal = hexToNumberString(tokenId);
+  return `${ETHERSCAN_BASE_URL}token/${registryAddress}?a=${tokenIdDecimal}`;
+};
+
+interface ERC721TokenParameters {
+  registryAddress: string;
+  tokenId: string;
+}
+
+export const AssetInfo = ({ registryAddress, tokenId }: ERC721TokenParameters) => {
```

Comment:
```
export const AssetInfo: FunctionComponent<ERC721TokenProps> = ({ registryAddress, tokenId }) => {
```

Can use FunctionComponent helper from React to type your component

Reply:
done